cypress/e2e/check-status.cy.ts

```diff
@@ -0,0 +1,120 @@
+/// <reference types="cypress" />
+
+export {};
+let nameFiles = [
+  '/',
+  'about',
+  'affiliates',
+  'annual',
+  'black-friday',
+  'byte-converter',
+  'cloud-storage-comparison',
+  'cloudwards',
+  'cyber-awareness',
+  'dealfuel',
+  'drive',
+  'freemonth',
+  'legal',
+  'lifetime_special',
+  'lifetime',
+  'media-area',
+  'open-source',
+  'partner-discount',
+  'password-checker',
+  'password-generator',
+  'photos',
+  'pricing',
+  'privacy-directory',
+  'privacy',
+  'stackcommerce',
+  'techradar-discount',
+  'temporary-email',
+  'virus-scanner',
+  'what-does-google-know-about-me',
+  'use-cases',
+  'internxt-for-digital-nomads',
+  'internxt-for-photographers',
+  'internxt-for-students',
+  'internxt-for-tech-companies',
+  'internxt-for-lawyers',
+  'internxt-for-journalists',
+];
+
+const langs = { en: 'en', es: 'es', de: 'de', fr: 'fr', it: 'it', ru: 'ru', zh: 'zh' };
+
+describe('Verify that pages do not produce a 500 error', () => {
+  nameFiles.forEach((route) => {
+    describe(`Verify that ${route} do not produce a 500 error`, () => {
+      describe('Verify the English version', () => {
+        it(`${route === '/' ? 'home' : route} page`, () => {
```

Comment:
Using conditionals on tests is commonly a bad practice. Why you do not have the route already set (like the array called `namedFiles`) so at least those conditionals disappear?

Reply:
I would iterate over languages rather than over pages, as `${route === '/' ? '' : route}` is not too clear when searching for the test that has failed.